src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public enum Color {
+    WHITE, GOLD, SILVER, ORANGE, GREY, LIGHTGREY, DARKGREY, BLACK;
+
+    public static Color getRandomColor() {
+        Random random = new Random();
+        return values()[random.nextInt(values().length)];
+    }
```

Comment:
logic from this method should be moved to ColorSupplier class

Reply:
done